build_wheel.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
+
+#************************************************#
+#           build_wheel.sh                       #
+#    Build a Python3 wheel for YOLACT            #
+#************************************************#
+
+REPO_NAME="yolact"
+NEEDED_FOLDERS=("data" "external" "layers" "scripts" "utils" "web")
```

Comment:
Just to set expectations appropriately, I had no idea this is how you make an array in bash.

For whatever reason, I tend to treat bash as a language of primitive types and whenever I get into a situation where I decide more complex types make sense, I  usually escalate to a scripting language like `ruby` or `python`.

That's not to say I don't use arrays in bash. I just do it differently. I don't have any reason to advocate for this style, so take it or leave it, but this is what I tend to do.

```bash
$ NEEDED_FOLDERS="data external layers scripts utils web"

# Let's start with the way you can shoot yourself in the foot. Don't do this.
# Unless it's what you mean to do.
$ for folder in "$NEEDED_FOLDERS"; do echo $folder; done
data external layers scripts utils web

# That's not what we wanted! If you quote the variable, `for` treats it as a single argument
# Which, actually, is normal `for` behavior:
$ for folder in "data external layers scripts utils web"; do echo $folder; done
data external layers scripts utils web

# With that out of the way, here's the actual way to do it

$ for folder in $NEEDED_FOLDERS; do echo $folder; done
data
external
layers
scripts
utils
web
```

🤷🏻‍♂️